### PR TITLE
Added DataService Collection from FDA

### PR DESCRIPTION
**Pull request by ekate**

### CODE REVIEW

File: Document.hs [hunk 1]
```diff
@@ -62,6 +62,7 @@ instance JSON.FromJSON Value where
   parseJSON (JSON.Bool True) = return $ value "true"
   parseJSON (JSON.Bool False) = return $ value "false"
   parseJSON (JSON.Array a) = Value . concatMap values <$> mapM JSON.parseJSON a
+  parseJSON (JSON.Null) = return $ value T.empty
```
Comment (dylex) [inline Document.hs:65]: I presume this helped with some parsing? I'd only left it out to avoid false positives.

Comment (ekate) [reply Document.hs:65]: yes, in the new collection when I request JSON I get {"key":"dc.identifier.citation","value":null,"language":"en_US"} (which should not usually happen but I guess might when people use FDA REST API to populate collections )
and it was giving me :
primo-endpoint: user error (Error in $.value: expected Value, encountered Null)

Comment (ekate) [reply Document.hs:65]: Also, I am not sure I understand what you mean under "false positives" ?

Comment (dylex) [reply Document.hs:65]: I just mean that I wanted to make sure there weren't any nulls that I was ignoring that I shouldn't be. You are here converting null to `[""]`, a single empty string, whereas I might instead convert it to `[]`, the empty list (i.e., `mempty` or `Value []`), but as long as it's doing what you want it's fine.

**Pull request merged**